src/extension.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from 'vscode';
+
+export function activate(context: vscode.ExtensionContext) {
+
+	// This line of code will only be executed once when your extension is activated
+	// console.log('Congratulations, your extension "vscode-epsilon" is now active!');
+
+	vscode.window.registerTerminalLinkProvider({
+		provideTerminalLinks: (context, token) => {
+            
+            // Regular expression that matches Epsilon program locations
+            let regexp = /\(((.*?)@(\d*):(\d*)-(\d*):(\d*))\)/i
```

Comment:
Does this match windows paths as well or only unix?

Reply:
It should but I haven't tried it on Windows.

Reply:
No worries. If not, we can fix it later.